.azure-pipelines/cpu-unit-test.yml

```diff
@@ -7,8 +7,8 @@ trigger:
 
 strategy:
   matrix:
-    python-3.6:
-      imageTag: '3.6'
+    python-3.8:
+      imageTag: '3.8'
```

Comment:
why it cannot run on python3.6... it may break existing pipelines which reply on ubuntu18 and python3.6

Reply:
Shall we just add 3.8? Not replace 3.6?

Reply:
updated to add 3.8  and keep 3.6 and 3.7

Reply:
the dependency is:

- python 3.6 only support numpy < 1.20
- latest numpy requires networkx >= 2.5
- latest networkx and later requires python3.8+
- latest pandas requires numpy >=1.20.3

so currently:
networkx >= 2.5
numpy>=1.19.2
pandas>=1.15

Reply:
Can we check it in setup.py?

Reply:
updated to install networkx,numpy,pandas according to python version